.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "ruby.rubocop.executePath": "/usr/local/Homebrew/Library/Homebrew/shims/gems/"
```

Comment:
Drawing attention to this because it's hardcoded to `/usr/local/Homebrew` at the moment. Not sure how to make this dynamic.